__tests__/endpoints/articles.test.js

```diff
@@ -0,0 +1,42 @@
+const { app, data, seed, db, request } = require("../../testImports");
+
+describe("/api/articles/:article_id", () => {
+	test("GET: 200 - Return an article object to the client using an ID param", () => {
+		return request(app)
```

Comment:
Solid test here, though a single article being returned shouldn't be returned in an array.
It would be a more flexible test if you used `toMatchObject` on the article object rather than the whole response body.